packages/dev/core/src/FrameGraph/frameGraphTextureManager.ts

```diff
@@ -0,0 +1,378 @@
+/* eslint-disable import/no-internal-modules */
+import type {
+    Scene,
+    AbstractEngine,
+    RenderTargetWrapper,
+    RenderTargetCreationOptions,
+    TextureSize,
+    Nullable,
+    FrameGraphTextureCreationOptions,
+    FrameGraphTextureHandle,
+} from "core/index";

```

Comment:
probably not the right path

Reply:
This works because this import only imports types. @ryantrem explained that this could be done, and I find it more readable this way: first you have a single import for all types, then all the normal imports of "non-types".

Reply:
cc @ryantrem and @RaananW in case you didn't see it.

Reply:
The problem is that babylonjs/index is not defined in UMD. I can look into that. Is it a breaker, or can it wait a bit?

Reply:
It can wait a few days, we are not going to merge this PR just now.

Reply:
Sorry I missed this. If we can figure out the UMD issue though, then I find this to be much cleaner. I didn't realize this would cause a problem for UMD.

Reply:
i'll pull this and build it locally

Reply:
Resolved